core/build.gradle

```diff
@@ -24,8 +24,15 @@ import org.elasticsearch.gradle.test.RestSpecHack
 apply plugin: 'elasticsearch.build'
 apply plugin: 'com.bmuschko.nexus'
 apply plugin: 'nebula.optional-base'
+apply plugin: "application"
 
 archivesBaseName = 'elasticsearch'
+def runEsHome="${project.buildDir}/run-elastic"
+mainClassName = "org.elasticsearch.bootstrap.Elasticsearch"
+applicationDefaultJvmArgs = ["-Xms256m", "-Xmx1g", "-Djava.awt.headless=true", "-Delasticsearch", "-Des.foreground=yes",
+                             "-ea", "-Des.path.home=" + runEsHome, "-Des.security.manager.enabled=false",
```

Comment:
I think its hard to reproduce the previous behavior without fixing https://github.com/elastic/elasticsearch/issues/14361 first, but afaik, @rjernst is working on that right now.

at a high level: thats' what the previous code did: invoke the "integration test setup" but passing "fork=false", passing different port numbers (the conventional 9200/9300), and passing some additional jvm args to enable the remote debugger.